pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/handler/EmailNotificationHandler.java

```diff
@@ -199,7 +237,9 @@ boolean messageShouldBeIgnored(Handle apisHandle, String studyGuid, String parti
         subs.put(DDP_PARTICIPANT_LAST_NAME, message.getParticipantLastName());
 
         for (NotificationTemplateSubstitutionDto sub : message.getTemplateSubstitutions()) {
-            subs.put(sub.getVariableName(), sub.getValue());
+            if (sub.getVariableName().startsWith("-")) {
```

Comment:
if we think we need to support legacy substitution variables that doesn't follow : "-ddp." , we can remove this check